src/Main.java

```diff
@@ -1,76 +1,94 @@
+import java.security.spec.RSAOtherPrimeInfo;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
-       //Задача 1
-        int age=20;
-        int oldAge=18;
-        if (age >= oldAge) {
-        System.out.println("Если возраст человека равен "+age+", то он совершеннолетний");
-        } else {
-        System.out.println("Если возраст человека равен"+age+" он не достиг совершеннолетия, нужно немного подождать");
+        //Задача 1
+        System.out.println("Задача 1");
+        int money = 15_000;
+        int allMoney = 2_459_000;
+        double sum = 0;
+        int month = 0;
+        while (sum <= allMoney) {
+            month++;
+            sum += money;
+            System.out.println("Месяц " + month + ", сумма накоплений равна " + sum + " рублей");
+
         }
         //Задача 2
-        int temprature=2;
-        int normal=5;
-        if (temprature<normal) {
-            System.out.println("На улице " + temprature + " градуса, нужно надеть шапку");
-        } else if(temprature>normal){
-            System.out.println("На улице " + temprature + " градуса можно идти без шапки");
+        System.out.println("Задача 2");
+        int a = 0;
+        int b = 10;
+        while (a < b) {
+            a++;
+            System.out.print(" " + a + " ");
+        }
+        System.out.println();
+        for (int c = 10; c > 0; c--) {
```

Comment:
```suggestion
        for (int c = b; c > a; c--) {
```
раз уж завёл переменные, то используй